kim_cole/Assignments/surveyform/app/controllers/surveys_controller.rb

```diff
@@ -0,0 +1,34 @@
+class SurveysController < ApplicationController
+
+	def new
+		if not session[:count]
+			session[:count] = 0
+		end
+		@languages = ['PHP', 'Ruby', 'JavaScript', 'Python', 'C#', 'Spanish']
+
+		@locations = ['Dallas', 'San Diego', 'Chicago', 'D.C.']
+	end
+
+	def create
+		survey = Survey.new(survey_params)
+		p survey.valid?
+		if not survey.valid?
```

Comment:
You can just use survey.save, since it calls valid? I would also avoid using not unless you need to. There's not reason it can't simply be if survey.save